provider/aplus.py

```diff
@@ -159,7 +162,7 @@ def load_exercise_template(exercise, config, invalidate_cache=False):
     return template_source
 
 
-def get_radar_config(exercise_data):
+def get_radar_config(exercise_data, course):
```

Comment:
In the function `provider.tasks.create_submission()`, this function `get_radar_config()` is called, but you have not updated the parameters in the call. The function already contains a `course` variable so adding it to the function call should be easy.